cmd/crank/beta/xpkg/init.go

```diff
@@ -116,14 +138,127 @@ func (c *initCmd) Run(k *kong.Context, logger logging.Logger) error {
 		return errors.Wrapf(err, "failed to get repository's HEAD from %q", repoURL)
 	}
 
-	// TODO(phisco): replace placeholders for the name all around the
-	// 	repository? Maybe we can just agree on some markdown text in the
-	// 	repos to print to let the user know what to do next?
-
 	_, err = fmt.Fprintf(k.Stdout, "Initialized package %q in directory %q from %s (%s)\n", c.Name, c.Directory, getPrettyURL(logger, repoURL, ref), ref.Name().Short())
+	if err != nil {
+		return errors.Wrap(err, "failed to write to stdout")
+	}
+
+	err = handleNotes(k.Stdout, logger, c.Directory)
+	if err != nil {
+		return errors.Wrap(err, "failed to handle NOTES.txt")
+	}
+
+	err = c.handleInitScript(k, logger, c.Directory, c.Name)
+	if err != nil {
+		return errors.Wrap(err, "failed to handle initialization script")
+	}
+
 	return err
 }
 
+// handleNotes prints the NOTES.txt file in the template
+// repository, if it exists.
+func handleNotes(w io.Writer, logger logging.Logger, dir string) error {
+	notesFile := dir + "/" + notes
```

Comment:
is https://pkg.go.dev/path/filepath#Join better to use here? same comment for the handleInitScript function too.